snapcraft/main.py

```diff
@@ -23,11 +23,13 @@
   snapcraft [options]
 
 Options:
-  -h --help        show this help message and exit
-  -v --version     show program version and exit
-  -V --verbose     print additional information about command execution
-  -d --debug       print debug information while executing (including
-                   backtraces)
+  -h --help              show this help message and exit
+  -v --version           show program version and exit
+  -V --verbose           print additional information about command execution
+  -d --debug             print debug information while executing (including
+                         backtraces)
+  --no-parallel-build    use only a single build job per part (defaults to
+                         number of CPUs)
```

Comment:
This default is confusing. A little. I think I need more words, but I can't find a good way to say it.
Something like: defaults to use the number of CPUs to build parts in parallel.
The problem with this one is that it sounds like you will be building many parts at the same time, instead of building one part at a time.